tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -495,12 +506,14 @@ TangoErrorType TangoRosNode::TangoSetupConfig() {
 TangoErrorType TangoRosNode::TangoConnect() {
   const char* function_name = "TangoRosNode::TangoConnect()";
 
-  TangoCoordinateFramePair pair;
-  pair.base = TANGO_COORDINATE_FRAME_START_OF_SERVICE;
-  pair.target = TANGO_COORDINATE_FRAME_DEVICE;
-
+  TangoCoordinateFramePair pairs[2] = {
+      { TANGO_COORDINATE_FRAME_START_OF_SERVICE,
+        TANGO_COORDINATE_FRAME_DEVICE }, {
+        TANGO_COORDINATE_FRAME_AREA_DESCRIPTION,
+        TANGO_COORDINATE_FRAME_DEVICE }
+  };
   TangoErrorType result;
-  result = TangoService_connectOnPoseAvailable(1, &pair, onPoseAvailableRouter);
+  result = TangoService_connectOnPoseAvailable(2, pairs, onPoseAvailableRouter);
```

Comment:
Well, this one was here before.. perhaps this magic number can be replaced with something more meaningful?